enterprise/coderd/groups.go

```diff
@@ -257,15 +266,16 @@ func (api *API) deleteGroup(rw http.ResponseWriter, r *http.Request) {
 		ctx               = r.Context()
 		group             = httpmw.GroupParam(r)
 		auditor           = api.AGPL.Auditor.Load()
-		aReq, commitAudit = audit.InitRequest[database.Group](rw, &audit.RequestParams{
+		aReq, commitAudit = audit.InitRequest[database.AuditableGroup](rw, &audit.RequestParams{
 			Audit:   *auditor,
 			Log:     api.Logger,
 			Request: r,
 			Action:  database.AuditActionDelete,
 		})
 	)
 	defer commitAudit()
-	aReq.Old = group
+	var emptyUsers []database.User
+	aReq.Old = group.Auditable(emptyUsers)
```

Comment:
shouldn't the `aReq.New` be `emptyUsers` and the `aReq.Old` have the members?

Reply:
Great catch @Emyrk! Updated.